### DISCUSSION TITLE
Is it required to keep widevinecdmadapter.dll in client application with CEF version 112

### DISCUSSION THREAD

**Discussion by susmitakanchinadham** [cefsharp/CefSharp#4545] [General]
We have recently updated to CEF v112 for our Desktop application.
Do we still need to maintain the widevinecdmadapter.dll with out app?
I see in the below issue that it is not required anymore. Can you please confirm.
https://github.com/cefsharp/CefSharp/issues/3767

**Comment by amaitland** [comment]
> #3767

I've updated the referenced link as the CEF issue tracker moved since it was originally created. You can read all about the changes yourself.

https://github.com/chromiumembedded/cef/issues/3149

**Reply by susmitakanchinadham** [reply]
@amaitland My actual query is, do we still need to maintain the widevinecdmadapter.dll with our application to support decryption of protected content from streaming services like youtube.

**Reply by amaitland** [reply]
https://github.com/chromiumembedded/cef/issues/3149 details how CEF now handles CDM.
